index.html

```diff
@@ -456,32 +421,171 @@ <h2 id="constraints">
         </p>
       </section>
       <section>
-        <h2 id="display-constraints">
-          New Constrainable Properties for Captured Display Surfaces
+        <h2 id="constrainable-properties">
+          Constrainable Properties for Captured Display Surfaces
         </h2>
-        <p>
-          Following new <a href="https://www.w3.org/TR/mediacapture-streams/#defining-a-new-constrainable-property">
-          Constrainable Properties</a> are defined that allow an application to observe the properties
-          of the selected <a>display surface</a>. Since the source of media cannot be changed after a
-          <code>MediaStreamTrack</code> has been returned and constraints do not affect the
-          selection of <a>display surfaces</a>, these constrainable properties cannot be changed by an
-          application.
-        </p>
-        <p>
-          The <code><a data-link-for="MediaTrackConstraintSet">displaySurface</a></code> constraint
-          allows an application to observe the type of <a>display surface</a> that is being
-          captured.
-        </p>
-        <p>
-          The <code><a data-link-for="MediaTrackConstraintSet">logicalSurface</a></code> constraint
-          allows an application to observe whether the surface that is captured is a <a>logical
-          display surface</a>, rather than the <a>visible display surface</a>.
-        </p>
-        <p>
-          The <code><a data-link-for="MediaTrackConstraintSet">cursor</a></code> constraint
-          allows an application to specify if the cursor should be included in the captured
-          display.
-        </p>
+        <p>Constraints serve a different purpose for display device capture. They
+        do not aid discovery, instead they are applied only after user-selection.
+        Some of these constraints enable user agent processing like downscaling
+        and frame decimation, as well as display-specific features. Others enable
+        observation of inherent properties of a user-selected
+        <a>display surface</a>, as capabilities and settings.</p>
+        <p>The following new and existing <code><a>MediaStreamTrack</a></code>
+        <a href="https://www.w3.org/TR/mediacapture-streams/#constrainable-properties">
+        Constrainable Properties</a> are defined to apply to the user-selected
+        video <a>display surface</a>, with the following behavior:</p>
```

Comment:
It would probably be worthwhile pointing out that the entries here, if they don't mention semantics as a setting, can't be used as one.

Reply:
Actually, they don't all mention "As a setting" but that doesn't mean they won't get once, since the constrainable pattern ensures you get all aspects of a constrainable property.

I tried to follow the pattern of [4.3.8 Constrainable Properties](https://www.w3.org/TR/mediacapture-streams/#constrainable-properties) in gUM, where we rely on an understanding of the constrainable pattern.

E.g. for `width` in gUM, all it says is *"The width or width range, in pixels. As a capability, the range should span the video source's pre-set width values with min being the smallest width and max being the largest width."*

It is therefore assumed that any constraint within min and max are valid, may get applied and reflected in *getSettings()*.

Similarly, any setting *not* in the list at all, does not count in the domain.

Do you still see the need for a change? If so, which specifically?

Reply:
Specifically, I don't think we can say *"if it's not on this list, then ignore it"* since that would lock out any extension spec ever.